plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py

```diff
@@ -57,6 +57,8 @@
 from aea.helpers import http_requests as requests
 from aea.helpers.base import try_decorator
 
+from .hashfuncs import ripemd160, sha256
```

Comment:
Avoid using relative imports in the framework

Reply:
@angrybayblade this is only one way, cause we have auto check for cosmos.py and _cosmos.py check

it will fail on packages names differ

Reply:
acually we have to reuse cosmos.py as imported from aea-ledger-cosmos, not copy it across plugins